apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/blocks/Card/BackgroundColor/BackgroundColor.tsx

```diff
@@ -132,6 +137,28 @@ export function BackgroundColor(): ReactElement {
     }
   }
 
+  function validateHexString(color: string): boolean {
```

Comment:
```suggestion
  function isValidHex(color: string): boolean {
```
would be common naming convention here